app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt

```diff
@@ -69,14 +105,25 @@ internal class FreshTab : StartTabFragment(), FreshTabContract.View {
         }
     }
 
-    override fun hideAllTrialPeriodViews() {
+    private fun hideAllTrialPeriodViews() {
         trial_period_lumen_upgrade.visibility = View.GONE
         trial_over_lumen_upgrade.visibility = View.GONE
     }
 
-    override fun toggleWelcomeMessage(show: Boolean) {
-        welcome_image.visibility = if (show) View.VISIBLE else View.GONE
-        welcome_title.visibility = if (show) View.VISIBLE else View.GONE
+    private fun toggleWelcomeMessage() {
+        welcome_image.visibility = if (isFreshInstall) View.VISIBLE else View.GONE
```

Comment:
Lets try to follow the [kotlin style guidelines](https://developer.android.com/kotlin/style-guide) by android
Use braces around single line if else

Reply:
It's an expression and fits in one line:
https://developer.android.com/kotlin/style-guide#expressions